phys2bids/viz.py

```diff
@@ -33,6 +36,15 @@ def plot_trigger(time, trigger, fileprefix, tr, thr, num_timepoints_expected,
     figsize: tuple
         Desired size of the figure (see `matplotlib`),
         Default is {FIGSIZE}
+    tr: float
+        Repetition time
+    thr:float
+        Threshold used to detect the number of triggers
+    num_timepoints_expected: int
+        Number of timepoints expected by the user
+    figsize: integer tuple or list
+        Size of the figure X*Y
```

Comment:
It's not clear to me whether you mean "integer, tuple or list" or a "tuple of integers or a list". Could you specify better and add those possible options in the entry above?

Also, I'm not totally sure it can/should be an integer. In theory, it could be afloat but it seems to me that it has to be a tuple (or a list, but matplotlib seems to reference a tuple).